mining.py

```diff
@@ -0,0 +1,78 @@
+import pickle
+import nltk
+import numpy
+from nltk.tokenize import TweetTokenizer
+import collections
+books = ['odyssey', 'iliad','bible', 'cant', 'shakes', 'don', 'pride', 'frank', 'alice', 'sherlock', 'wilde']
+unknown = 'suprise'
+
+def openFile(name):
+    pickledName = name + '.pickle'
+    opened = open(pickledName, 'rb')
+    txt = pickle.load(opened)
+    txt = txt.lower()
+    txt = txt.strip(',.?/1234567890~*()')
+    return txt
+
+def sortText(t):
+    tknzr = TweetTokenizer()
+    token = tknzr.tokenize(t)
+    return token
+
+def wordFreq(s):
+    c = collections.Counter()
+    for word in s:
+        c[word] += 1
+    return c
+
+def makeWords():
+    count = []
+    dictionaries = []
+    i = 0
+    while i < len(books):
+        sText = sortText(openFile(books[i]))
+        words = wordFreq(sText)
+        dictionaries.append(words)
+        length = len(sText)
+        count.append(length)
+        i+=1
+    return (dictionaries, count)
+
+def fgivenWord(word, dictionaries, count, u, ulen):
+    i = 0
```

Comment:
You should include a docstring comment for each function that describes what each function does, what input it expects, and what kind of output it produces.